grapher/mapCharts/MapChart.tsx

```diff
@@ -326,13 +326,17 @@ export class MapChart
         const { mapConfig, mapColumn, colorScale } = this
 
         return (d: PrimitiveType): string => {
-            if (isString(d)) return d
-            else if (mapConfig.tooltipUseCustomLabels) {
+            if (mapConfig.tooltipUseCustomLabels) {
                 // Find the bin (and its label) that this value belongs to
                 const bin = colorScale.getBinForValue(d)
                 const label = bin?.label
                 if (label !== undefined && label !== "") return label
             }
+            else if (isString(d)) {
+                const bin = colorScale.getBinForValue(d)
+                const label = bin?.label
+                if (label !== undefined && label !== "") return label
```

Comment:
If we're in this block (i.e. the datum is a string and the "Use custom tooltip labels" checkbox is _not_ checked), we simply want to return the string value as-is:

```suggestion
                return d
```

Reply:
I have applied the change, please let me know if I should change anything else.